satpy/tests/test_scene.py

```diff
@@ -595,6 +595,16 @@ def test_coarsest_finest_area(self):
         assert scene.finest_area() is area_def2
         assert scene.coarsest_area(['2', '3']) is area_def2
 
+        # test logic for flipped areas
+        area_def1_flipped = area_def1.copy(area_extent=tuple([-1*ae for ae in area_def1.area_extent]))
+        area_def2_flipped = area_def2.copy(area_extent=tuple([-1*ae for ae in area_def2.area_extent]))
+        ds1.attrs['area'] = area_def1_flipped
+        ds2.attrs['area'] = area_def2_flipped
+        ds3.attrs['area'] = area_def2_flipped
+        assert scene.coarsest_area() is area_def1_flipped
+        assert scene.finest_area() is area_def2_flipped
+        assert scene.coarsest_area(['2', '3']) is area_def2_flipped
```

Comment:
Would it be possible for you to turn `test_finest_coarsest_area` into its own test case class, containing two methods, one for the regular case and one for the flipped case? I think it would make the tests a bit cleaner. But if you don't have time, we can also merge it like this.

Reply:
Thanks for the review. Just added a commit, does it make sense like this?